src/lib/Commands/TestCommand/index.js

```diff
@@ -337,10 +388,18 @@ ${bootstrapCode}
 }
 
 // resolve modules
-__module_tests(__module_ImpUnit_Promise_exports, __module_impUnit_exports.ImpTestCase);
```

Comment:
The one issue can be -> the isPending() method was in old (removed) Promise AND It is absent in promise.class.nut:3.0.0